package.json

```diff
@@ -13,7 +12,15 @@
     "deploy-rules-production": "firebase use production; firebase deploy --only database"
   },
   "dependencies": {
-    "nodemon": "^1.11.0"
+    "body-parser": "^1.17.2",
+    "cors": "^2.8.4",
+    "dotenv": "^4.0.0",
```

Comment:
@hugurp with Firebase functions, do we use the `.runtimeconfig.json` file instead of `dotenv`?

 If so, let's write it in readme and remove `dotenv`.

Reply:
ok